build_tools/github/build_minimal_windows_image.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+set -x
+
+PYTHON_VERSION=$1
+BITNESS=$2
+
+if [[ "$PYTHON_VERSION" == "36" || "$BITNESS" == "32" ]]; then
+    # Python 3.6 and 32-bit architectures are not
+    # yet supported by the official Docker images
+    exit 0
+fi
+
+if [[ "$PYTHON_VERSION" == "37" ]]; then
+    WHEEL_NAME=cp"$PYTHON_VERSION"m-win_amd64.whl
+else
+    # Different name for Python 3.8 and newer
+    WHEEL_NAME=cp"$PYTHON_VERSION"-win_amd64.whl
+fi
+
+WHEEL_NAME=scikit_learn-$SCIKIT_LEARN_VERSION-cp$PYTHON_VERSION-$WHEEL_NAME
+
+# Find the repaired wheel because there is no a way
+# to access to the path in a straightforward manner
+WHEEL_PATH="C:/Users/RUNNER~1/AppData/Local/Temp"
```

Comment:
If the following works, I think it's a better option (in case `RUNNER~1` gets renamed for instance):

```suggestion
WHEEL_PATH="$HOME/AppData/Local/Temp"
```